OpenRA.Game/World.cs

```diff
@@ -113,9 +121,13 @@ public void SetLocalPlayer(string pr)
 			RenderPlayer = LocalPlayer;
 		}
 
-		public readonly Actor WorldActor;
-		public readonly Map Map;
-		public readonly TileSet TileSet;
+		public Actor WorldActor { get; private set; }
```

Comment:
Same as my former comment about public readonly fields. If my architect boss ever heards I do a public readonly fields I'd get fired for sure :P